Source/Core/DolphinWX/Config/WiiConfigPane.cpp

```diff
@@ -21,6 +21,18 @@
 #include "DolphinWX/WxUtils.h"
 #include "UICommon/USBUtils.h"
 
+// SYSCONF uses 0 for bottom and 1 for top, but showing top first in the GUI
+// makes the most sense, so we number them the other way in the GUI code
```

Comment:
This seems like a very opinionated reason to move them around for. You might want to replace it with some more concrete reasoning (the Wii also places the Above TV option above the Below TV option) rather than implying "it makes sense because it does".